<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtPrintAndPreview</class>
 <widget class="QDialog" name="BtPrintAndPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print And Preview</string>
  </property>
  <property name="configSection" stdset="0">
   <string notr="true">btPrintAndPreview</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTabWidget" name="verticalTabWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="locale">
              <locale language="English" country="UnitedStates"/>
             </property>
             <property name="tabPosition">
              <enum>QTabWidget::West</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="printAndPreviewTabRecipe">
              <attribute name="title">
               <string>Recipe</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="topMargin">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Select Recipe(s)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_SelectedRecipe"/>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="verticalGroupBox_3">
                   <property name="title">
                    <string>Select what to include</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <property name="leftMargin">
                     <number>10</number>
                    </property>
                    <item>
                     <widget class="QCheckBox" name="checkBox_Recipe">
                      <property name="text">
                       <string>Recipe</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkBox_2">
                      <property name="text">
                       <string>Brewday Instructions</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="printAndPreviewTabInventory">
              <attribute name="title">
               <string>Inventory</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7"/>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Select printer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Select available Printers</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Printer Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="Button_PrinterSettings">
               <property name="toolTip">
                <string>Printer Settings</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2"/>
           </item>
           <item>
            <widget class="QGroupBox" name="horizontalGroupBox_2">
             <property name="title">
              <string>Outout</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkBox_4">
                <property name="text">
                 <string>Paper</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_3">
                <property name="text">
                 <string>PDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>HTML</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_Orientation">
             <property name="title">
              <string>Orientation</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QRadioButton" name="radioButton_Protrait">
                <property name="text">
                 <string>Portrait</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_Landscape">
                <property name="text">
                 <string>Landscape</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_PrintPreviewWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Print">
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../brewtarget.qrc"/>
 </resources>
 <connections/>
</ui>
